### PR TITLE
Docs for noisy tournaments

### PR DESCRIPTION
**Pull request by marcharper**
Re: [Issue #206](https://github.com/Axelrod-Python/Axelrod/issues/206)

### CODE REVIEW

**Comment by marcharper** [comment]
I guess github treats SVG as text files so the line count is huge. We can switch to e.g. PNG if this bothers anyone (I don't care about the line count either way).

File: docs/usage.rst [hunk 1]
```diff
@@ -242,6 +242,86 @@ Thus we see that there are multiple Nash equilibria for this game. Two pure equi
 
 To further study how this system evolves over time and how robust some of the observations we have made are let us look at how this game can be interpreted in an ecological setting.
```
Comment (drvinceknight) [inline docs/usage.rst]: This paragraph introduces the ecological tournament. Could you move this to the end of the section on noisy tournaments and then add a sentence here to replace it which introduces noisy tournaments?

**Comment by drvinceknight** [comment]
This is looking really awesome :+1: I've made that one comment, about moving an introductory sentence, let me know what you think :)

**Comment by drvinceknight** [comment]
I have no problem with the svg files. I can't remember why I went with png for the original ones... Perhaps we could change those to svg as well? (Not for this PR)

**Comment by marcharper** [comment]
I moved that sentence to the beginning of the ecological section and left the rest as is. The first paragraph in the noisy tournaments section tries to motivate, but if someone else has a better intro by all means please update the section.

I had to search for a bit to find out how to make the images line up side-by-side in rst. It is a nice trick!

File: README.rst [hunk 2]
```diff
@@ -114,7 +114,7 @@ To write/render the documenation locally, you will need
 
 ::
 
-    $ pip install sphinx sphinx-autobuild
+    $ pip install sphinx sphinx-autobuild mock
```
Comment (drvinceknight) [inline README.rst]: What is the mock for? (Sorry only just noticed this...)

**Comment by marcharper** [comment]
I got an error when I tried to build the docs (on Ubuntu) because I did not have mock installed.

**Comment by marcharper** [comment]
Here's the error (I just tried making the docs in a clean virtual machine image):

user@kubuntu-vbox:~/Axelrod/docs$ make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.3.1
making output directory...

Exception occurred:
  File "conf.py", line 18, in <module>
    import mock
ImportError: No module named mock
The full traceback has been saved in /tmp/sphinx-err-ahHO6O.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:53: recipe for target 'html' failed
make: *** [html] Error 1

If I run
> sudo pip install mock

Then "make html" works fine.

**Comment by drvinceknight** [comment]
Cool :)

**Pull request merged**